core-bundle/templates/Error/backend.html.twig

```diff
@@ -1,9 +1,8 @@
-{% trans_default_domain 'contao_exception' %}
-{% extends "@ContaoCore/Error/layout.html.twig" %}
-{% block title %}
-    {{ statusName }}
-{% endblock %}
-{% block main %}
-    <h2>{{ 'XPT.matter'|trans }}</h2>
-    <p>{{ exception }}</p>
-{% endblock %}
+{% extends "@Contao/error/backend_fallback.html.twig" %}
+
+{#
+    ! DEPRECATED !
+
+    The "@ContaoCore/error/backend.html.twig" template will be removed in Contao 6.
+    Use the "@Contao/error/backend.html.twig" template from the @Contao namespace instead.
```

Comment:
However, the is only a `backend_fallback.html.twig` template 🤔

Reply:
The way the pretty errors are currently rendered, it would need some refactoring if we would only (fully) move some of them. That's why I kept all of them.

Or do we maybe want to move _all_ error templates already?

Reply:
You are missing my point. The comment says `Use the "@Contao/error/backend.html.twig" template` but such a file does not exist. There is only a `backend_fallback.html.twig` file. Do we have to rename this file to `backend.html.twig`?

Reply:
Ah, got you. :man_facepalming:. Yes, the commen must be adjusted, fixed in 196dc1d57b00c2028e34e2cdb3c99f06d1d8125c.

Reply:
No, the comment is correct IMHO. The template needs to be renamed to `core-bundle/contao/templates/twig/error/backend.html.twig`, because the `_fallback` suffix does not make any sense to me.

Reply:
Hmm, I can't remember the reasoning for that anymore. Switched it to `backend` like you suggested in 64f54a55da80cee9e2382be9e8911ecbf27572e5.